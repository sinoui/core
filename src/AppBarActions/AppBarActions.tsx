import styled from 'styled-components';

const AppBarActions = styled.div`
  display: flex;
  align-items: center;
  margin-left: 32px;
  & > *:not(:last-child) {
    margin-right: 24px;
  }
`;

export default AppBarActions;
