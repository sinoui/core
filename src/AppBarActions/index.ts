import AppBarActions from './AppBarActions';

export default AppBarActions;
