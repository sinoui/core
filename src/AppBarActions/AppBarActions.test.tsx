import React from 'react';
import { ThemeProvider } from 'styled-components';
import { defaultTheme } from '@sinoui/theme';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import AppBarActions from '@sinoui/core/AppBarActions';

/**
 * 顶部应用栏 AppBarActions组件 单元测试
 */

it('AppBarActions', () => {
  const tree = renderer
    .create(
      <ThemeProvider theme={defaultTheme}>
        <AppBarActions>icons</AppBarActions>
      </ThemeProvider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
