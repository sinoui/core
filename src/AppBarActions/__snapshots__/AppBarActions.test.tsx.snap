// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBarActions 1`] = `
<div
  className="AppBarActions-sl4kx3-0 iaQAlT"
>
  icons
</div>
`;
